### PR TITLE
add newrelic-prometheus-configurator

### PR DESCRIPTION
**Pull request by developer-guy**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [x] REQUIRED - The package is added to `packages.txt`

#### For security-related PRs

- [ ] The security fix is recorded in `advisories` and `secfixes`

#### For version bump PRs

- [ ] The `epoch` field is reset to 0
- [ ] Patch source: _patch source here_

### CODE REVIEW

File: newrelic-prometheus-configurator.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/newrelic/newrelic-prometheus-configurator
+      tag: v${{package.version}}
+      expected-commit: df4034f801f4f254243a38b110cb1b0af6b0aca5
+
+  - runs: |
+      make GOOS=$(go env GOOS) GOARCH=$(go env GOARCH) build
+      install -Dm755 ./bin/prometheus-configurator "${{targets.destdir}}"/usr/bin/ctop
```
Comment (ajayk) [inline newrelic-prometheus-configurator.yaml]: what is ctop here

**Review by ajayk — CHANGES_REQUESTED** [review]

File: newrelic-prometheus-configurator.yaml (hunk 1 above)
Comment (developer-guy) [reply newrelic-prometheus-configurator.yaml]: fixed

File: newrelic-prometheus-configurator.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - automake
```
Comment (ajayk) [inline newrelic-prometheus-configurator.yaml:17]: `automake`  not needed here 
cc: @kaniini

Comment (developer-guy) [reply newrelic-prometheus-configurator.yaml]: yes you are right, removed!

**Review by ajayk — APPROVED** [review]

**Pull request merged**